x-pack/plugins/security_solution/public/management/components/running_processes_action_results/running_processes_action_results.tsx

```diff
@@ -0,0 +1,285 @@
+          <SentinelOneRunningProcessesResults
+            action={action}
+            agentId={agentId}
+            data-test-subj={dataTestSubj}
+          />
+        ) : null}
+      </EuiText>
+    );
+  }
+);
+RunningProcessesActionResults.displayName = 'RunningProcessesActionResults';
+
+// @ts-expect-error TS2769
+const StyledEuiBasicTable = styled(EuiBasicTable)`
+  table {
+    background-color: transparent;
+    font-size: inherit;
+  }
+
+  .euiTableHeaderCell {
+    border-bottom: ${(props) => props.theme.eui.euiBorderThin};
+
+    .euiTableCellContent__text {
+      font-weight: ${(props) => props.theme.eui.euiFontWeightRegular};
+    }
+  }
+
+  .euiTableRow {
+    &:hover {
+      background-color: ${({ theme: { eui } }) => eui.euiColorEmptyShade} !important;
+    }
+
+    .euiTableRowCell {
+      border-top: none !important;
+      border-bottom: none !important;
+    }
+  }
+`;
+
+interface EndpointRunningProcessesResultsProps {
+  action: MaybeImmutable<ActionDetails<GetProcessesActionOutputContent>>;
+  /** If defined, only the results for the given agent id will be displayed. Else, all agents output will be displayed */
+  agentId?: string;
+  'data-test-subj'?: string;
+}
+
+/** @private */
+const EndpointRunningProcessesResults = memo<EndpointRunningProcessesResultsProps>(
+  ({ action, agentId, 'data-test-subj': dataTestSubj }) => {
+    const testId = useTestIdGenerator(dataTestSubj);
+    const agentIds: string[] = agentId ? [agentId] : [...action.agents];
+    const columns = useMemo(
+      () => [
+        {
+          field: 'user',
+          'data-test-subj': testId('user'),
+          name: i18n.translate(
+            'xpack.securitySolution.endpointResponseActions.getProcesses.table.header.user',
+            { defaultMessage: 'USER' }
+          ),
+          width: '10%',
+        },
+        {
+          field: 'pid',
+          'data-test-subj': testId('pid'),
+          name: i18n.translate(
+            'xpack.securitySolution.endpointResponseActions.getProcesses.table.header.pid',
+            { defaultMessage: 'PID' }
+          ),
+          width: '5%',
+        },
+        {
+          field: 'entity_id',
+          'data-test-subj': testId('entity_id'),
+          name: i18n.translate(
+            'xpack.securitySolution.endpointResponseActions.getProcesses.table.header.enityId',
+            { defaultMessage: 'ENTITY ID' }
+          ),
+          width: '30%',
+        },
+
+        {
+          field: 'command',
+          'data-test-subj': testId('command'),
+          name: i18n.translate(
+            'xpack.securitySolution.endpointResponseActions.getProcesses.table.header.command',
+            { defaultMessage: 'COMMAND' }
+          ),
+          width: '55%',
+        },
+      ],
+      [testId]
+    );
+
+    const wrappingClassname = useMemo(() => {
+      return css({
+        '.accordion-host-name-button-content': {
+          'font-size': 'inherit',
```

Comment:
```suggestion
          fontSize: 'inherit',
```
else you get this error on console
![Screenshot 2024-07-30 at 15 49 05](https://github.com/user-attachments/assets/82057da3-d97d-457f-a105-2a0906906abd)

Reply:
Thanks. I don't remember getting this warning 🤔 

When you made the change, did you check the UI to ensure it was picking up the correct font size? (no worries if you did not... I can test it out)

Reply:
yeah the font size was correct with the change. 
![Screenshot 2024-07-30 at 16 08 40](https://github.com/user-attachments/assets/2f91a6ea-30e4-4613-a4ac-47bb6b10af84)

Reply:
cool. thanks. 
I'll make this change in the next PR - since I got all approvals here and a green build. Hope that is cool with you.